ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift

```diff
@@ -67,7 +67,7 @@ public class GatewayAPI: NSObject, NSCoding {
             return
         }
 
-        let requestURL = "\(self.gatewayAddress.absoluteString)/\(self.app.siteName)/token"
+        let requestURL = "\(self.gatewayAddress.absoluteString!)/\(self.app.siteName)/token"
```

Comment:
make `self.gatewayAddress.absoluteString!` computed property so it will be more elegance.

```swift
private getAddressAbsoluteString() -> String { return self.gatewayAddress.absoluteString!}
```